.circleci/config.yml

```diff
@@ -10,6 +10,77 @@ jobs:
           command: echo "Hello world!"
     working_directory: /usr/src/artman/
 
+  build_and_push_docker_image:
+    working_directory: /usr/src/artman
+    docker:
+      - image: docker:17.05.0-ce-git
+    branches:
+      only:
+        - master
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Fingers crossed!
+          command: |
+            echo "Fingers crossed!"
+      - restore_cache:
```

Comment:
Unless you get a substantial speed improvement, I would probably omit this (and `save_cache`).